### PR TITLE
Bar is hiding in case of iPhone X.

### LINKED ISSUE #2: Bar is hiding in case of iPhone X
screen shot 2018-05-10 at 3 55 33 pm

### PR DESCRIPTION
**Pull request by maroofkay**
Resolve #2

### CODE REVIEW

File: Source/NotificationBarConfiguration.swift [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class NotificationBarConfiguration {
         duration = 4.0
         font = UIFont.systemFont(ofSize: 16.0)
         textColor = .white
-        padding = 4.0
+        padding = 1.0
```
Comment (furqanmk) [inline Source/NotificationBarConfiguration.swift]: What's the reason behind this change?

Comment (maroofkay) [reply Source/NotificationBarConfiguration.swift]: The idea of padding has changed. This is now just bottom padding.

Comment (furqanmk) [reply Source/NotificationBarConfiguration.swift]: Should we rename it to `bottomPadding` then?

Comment (maroofkay) [reply Source/NotificationBarConfiguration.swift]: We should.

**Review by furqanmk — APPROVED** [review]
